src/Action.ts

```diff
@@ -10,10 +10,10 @@ export class Action {
     let filtered = pullRequests.filter((pr) => pr.user?.login == this.userName)
 
     if (filtered.length > 1){
-      let newest = filtered[0].head.ref
       filtered.splice(0, 1)
       for (let i = 0; i < filtered.length; i++){
-        this.api.createComment(filtered[i].number, `Closing PR as a newer one from ${this.userName} exists at ${newest}`)
```

Comment:
I would just wrap this entire block in a try-catch. i.e if any request fails for each PR skip it and move on to the next one.

Reply:
Solid suggestion, I've updated and wrapped the calls in a single try/catch.